docs/querying/operators.md

```diff
@@ -211,6 +211,9 @@ all other labels are preserved the output. `by` does the opposite and drops
 labels that are not listed in the `by` clause, even if their label values are
 identical between all elements of the vector.
 
+`label_list` is a list of unquoted labels that may include a trailing comma, i.e.
```

Comment:
`label list` is what we have above.

Reply:
I've reordered the paragraphs to a more logical order to address this.